lib/puppet/type/scheduled_task.rb

```diff
@@ -140,6 +140,8 @@ def insync?(current)
           * `minutes_interval` --- The repeat interval in minutes.
           * `minutes_duration` --- The duration in minutes, needs to be greater than the
             minutes_interval.
+          * `disable_synchronise` --- Whether or not to disable the `synchronise across
```

Comment:
I would be tempted to name it something a bit more in line with the Windows functionality it's controlling. Perhaps `time_zone_synchronization`, or better still, exactly as it is in the UI: `synchronize_across_time_zones`. I will admit that's a bit long, but it will make it obvious what it's related to.

Reply:
Changed to `disable_time_zone_synchronization`. `synchronize_across_time_zones` doesn't really work due to the function turning it of rather than varying it or turning it on.